packages/python/plotly/plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -13,6 +13,15 @@
 import warnings
 
 
+# FIXME: don't test with vaex if vaex isn't installed
+try:
+    import vaex
```

Comment:
why not add it as an import to [these files](https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/test_requirements/requirements_311_optional.txt)?

Reply:
If that doesn't work, you could use the [optional imports](https://github.com/plotly/plotly.py/blob/3d36f1468f79277d483151fd1af9b25d7bf30de6/packages/python/plotly/plotly/figure_factory/_table.py#L4) `get_module` function to check for this.

Reply:
nice catch - I'll use the optional imports trick.